dev-requirements.txt

```diff
@@ -1,6 +1,10 @@
 mock
 httpretty
+splinter
```

Comment:
These were the ones in the .travis.yml before you made any changes to pytest, didn't notice your updates.